sensor/kubernetes/clustermetrics/cluster_metrics_test.go

```diff
@@ -62,10 +63,53 @@ func (s *ClusterMetricsTestSuite) TestMultipleNodes() {
 	s.Equal(expected, metrics)
 }
 
+func (s *ClusterMetricsTestSuite) TestOfflineMode() {
+	states := []common.SensorComponentEvent{
+		common.SensorComponentEventCentralReachable,
+		common.SensorComponentEventOfflineMode,
+		common.SensorComponentEventCentralReachable,
+	}
+	metrics := s.createNewClusterMetrics(time.Millisecond)
+	s.Require().NoError(metrics.Start())
+	defer metrics.Stop(nil)
+	for _, state := range states {
+		metrics.Notify(state)
+		s.assertOfflineMode(state, metrics)
+	}
+}
+
+func (s *ClusterMetricsTestSuite) createNewClusterMetrics(interval time.Duration) *clusterMetricsImpl {
+	metricsComponent := New(s.client)
+	metrics, ok := metricsComponent.(*clusterMetricsImpl)
+	s.Require().True(ok, "New should return a struct of type *clusterMetricsImpl")
+	metrics.pollingInterval = interval
+	return metrics
+}
+
+func (s *ClusterMetricsTestSuite) assertOfflineMode(state common.SensorComponentEvent, metrics *clusterMetricsImpl) {
+	switch state {
+	case common.SensorComponentEventCentralReachable:
+		select {
+		case <-time.After(10 * time.Second):
+			s.Fail("timeout waiting for the pollTicker to tick")
+		case <-metrics.pollTicker.C:
+			return
+		}
+	case common.SensorComponentEventOfflineMode:
+		select {
+		case <-time.After(10 * time.Millisecond):
```

Comment:
`1.01 * defaultInterval` would do the job, but `2` is the first multiplier that guarantees the result to be an int.
```suggestion
		case <-time.After(2*defaultInterval):
```

or as a minimum:

```suggestion
		case <-time.After(defaultInterval):
```

Reply:
Done